cmd/wallet-web/src/store/modules/options.js

```diff
@@ -30,6 +30,7 @@ let defaultAgentStartupOpts = {
     edvVaultID: '',
     edvCapability: '',
     authzKeyStoreURL: '',
+    kmsType: `local`,
```

Comment:
I replaced this inside keyServer to include the key servers URLs as well.. aren't these needed for onboarding?

Reply:
That will come from booststrap data. This is in web-wallet.

Reply:
https://github.com/trustbloc/edge-agent/pull/556/files#diff-2789e910946311a4b858c2162a7e018998c3aeddad0b308f6f28403884ce6346R85-R88